### ISSUE TITLE
Remote mock server

### ISSUE THREAD

**Issue by batuhansener** [DiUS/pact-consumer-swift#119]
Hi, are there any ways to run the mock service provider on a remote server so that anyone working the project will run their test connecting to that server and not being have to install pact-ruby-standalone?

**Comment by surpher** [comment]
Yes. When setting up your `MockService` object in your tests, you can set a baseURL for your remote server instead of `localhost` (or `127.0.0.1`).

Although it might be a bit tricky with the way `pact-ruby-standalone` works, if you would choose to use it as your mock server. #98 I personally haven't tried to set it up, but am very curious what you find out.
